tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.cs

```diff
@@ -60,20 +64,12 @@ internal readonly partial struct SecurityCoordinator
 
             _security.ApiSecurity.ShouldAnalyzeSchema(lastWafCall, _localRootSpan, args, _httpTransport.StatusCode.ToString(), _httpTransport.RouteData);
 
-            if (additiveContext != null)
-            {
-                // run the WAF and execute the results
-                if (runWithEphemeral)
-                {
-                    result = additiveContext.RunWithEphemeral(args, _security.Settings.WafTimeoutMicroSeconds, isRasp);
-                }
-                else
-                {
-                    result = additiveContext.Run(args, _security.Settings.WafTimeoutMicroSeconds);
-                }
+            // run the WAF and execute the results
+            result = runWithEphemeral
```

Comment:
Is it safe to assume `additiveContext` is _not_ null then, seeing as we removed the check?

Reply:
it's tested right [above l.60,](https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.cs#L60)  I think it was a refactoring that didn't remove this